Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
Frankly it's tempting to ship this with the existing `npm` and keep working on it without the time pressure of people being like _"Need an updated Node now"_.